README.md

```diff
@@ -12,28 +12,38 @@ We plan to release this plugin every time an OpenProject core release is done.
 **Beware**:
 
 * when translating `general_lang_name` do not translate the word 'English', but fill in the name of the language you are currently translating
-* This plugin adds translations for the OpenProject core, but not for plugins. But it provides a rake task to update the locales of plugins translated on Crowdin, namely `rake translations_for_plugins:update`. This rake task takes its configuration from a file. See `configuration.yml.example` for an example. 
 
-## Requirements
+## Technical Details
 
-* OpenProject version **3.0.0 or higher** ( or a current installation from the `dev` branch)
+We can split this up in three parts: storing locales, loading locales,
+updating locales.
 
-## Plugin Installation
+### Storing locales
+
+* Core: The locales (except for english) are stored in
+  OpenProject-Translations
+* Plugins: All locales are stored in the plugins themselves.
+
+### Loading Locales
 
-Edit the `Gemfile.plugins` file in your openproject-installation directory to contain the following lines:
+* Rails locales are loaded automatically. OpenProject-Translations just
+  patches the location Rails looks for locales.
+* Js locales get required by webpack during the rake task
+  `assets:webpack`. Currently, js locales from the core are required by
+OpenProject-Translations and js locales from plugins are required by
+each plugin individually.
 
-<pre>
-gem "openproject-translations", :git => 'https://github.com/opf/openproject-translations.git', :branch => 'stable'
-</pre>
+### Updating Locales
 
-Then update your bundle with the following commands executed from your openproject directory:
+* This plugin provides several rake tasks to update locales. These tasks keep the following up to date: the source files in Crowdin and the translated in the GitHub repos. We execute these tasks once a day.
```

Comment:
Maybe a typo "the source files in Crowdin and the *translations* in the Github repos."?